src/plugins/data/server/ui_settings.ts

```diff
@@ -265,7 +267,7 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
     },
     [UI_SETTINGS.SEARCH_INCLUDE_FROZEN]: {
       name: 'Search in frozen indices',
-      description: `Will include <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/frozen-indices.html"
+      description: `Will include <a href=${docLinks.links.elasticsearch.frozenIndices}
```

Comment:
FYI: This URL isn't particularly helpful, since it seems like frozen indices were deprecated in 7.14, but I've added that URL to the link service nonetheless.